### DISCUSSION TITLE
Signal Strength is weaker than other libraries

### DISCUSSION THREAD

**Discussion by mfvmx** [jgromes/RadioLib#489] [General]
We have an issing the RadioLib and max DB of 22 using a Dorji SX1262T module We are getting about 70% of the range as with the Stuart SX126XLT library, any thoughts on way to increase the power or reasons it doesn't match? Tested with  a box that was 1350ft away showning 90 RSSI with Stuart would not show up at all until we drove about half the distance closer to the device. Same device flashed with either library.
https://github.com/StuartsProjects/SX12XX-LoRa
http://www.dorji.com/products-detail.php?ProId=65
state = lora_modem.begin(FREQUENCY_CENTER_TRACK_STATERL, 500.0, 7, 5, SX126X_SYNC_WORD_PRIVATE, 22, 8, 3.3, false);
PowerDB       22
Preamble      8
TXCOVolt      3.3
RegLDO       false

**Comment by jgromes** [comment]
Since you haven't shared your code, I'm going to have to guess.

By default, RadioLib sets a 60 mA current limiter in the device (see [the wiki](https://github.com/jgromes/RadioLib/wiki/Default-configuration#sx126x---lora-modem)). According to the SX1262 datasheet, when transmitting at +22 dBm, the current draw will be something like 115 - 120 mA, so I suggest increasing the limit to the maximum of 140 mA with `setCurrentLimit(140.0)`.

**Reply by vtjballeng** [reply]
@jgromes , that made a big difference! It also looks like the SX1262 errata 15.1 & 15.2 in the Semtech datasheet is addressed in this library. 

I'd like to send code over, what's the best way to send it privately? Alternatively, you can email me at this username at gmail.com .

**Reply by jgromes** [reply]
If you have something that can't be shared publicly, send it to the email listed in library.properties file. Though keep in mind that I prefer all code to be shown publicly since someone else might run into the same issue as you.

**Comment by mfvmx** [comment]
That seems to do the track, thanks.
